modules/gdscript/gdscript_highlighter.cpp

```diff
@@ -294,6 +304,12 @@ void GDScriptSyntaxHighlighter::_update_cache() {
 	function_color = text_editor->get_color("function_color");
 	number_color = text_editor->get_color("number_color");
 	member_color = text_editor->get_color("member_variable_color");
+
+#ifdef TOOLS_ENABLED
```

Comment:
Do we need to compile any SyntaxHighlighter when tools are disabled?

Reply:
I guess not. It's only created when tools are enabled. I'll update with it excluded from scons.